scripts/plugins/caching-config.js

```diff
@@ -0,0 +1,23 @@
+export const Config = {
+  headers: { "hasura-m-auth": "zZkhKqFjqXR4g5MZCsJUZCnhCcoPyZ" },
+  time_to_live: 600,
+  redis_url: "redis://redis:6379",
+  otel_endpoint: "http://jaeger:4318/v1/traces",
+  otel_headers: {},
+
+  queries_to_cache: [
+    `
+      query MyQuery {
```

Comment:
Does this mean that only this query will cache? Including the query signature 'MyQuery' etc?

Reply:
At the moment, yes - whitespace/formatting doesn't matter, but everything else does. We can tailor this, though - e.g. we can remove `OperationName` as a factor. IIRC, the reason it's there is to allow users to bypass caching for now. (Also, I guess most of this will be redundant if we implement model-level caching)